ui/js/component/iconDownloaded/view.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Icon } from "component/common.js";
+
+const IconDownloaded = props => {
```

Comment:
Technically currently applies to both published stuff and downloaded stuff. If that's what we want, we could rename this to `<IconMyFile />` or `<IconMine />` or something like that. But I'm wondering if we should have a separate icon for published files anyway?